### PR TITLE
Newdle title cannot be more than 80 chars

### LINKED ISSUE #120: Limit newdle titles to a decent length

### PR DESCRIPTION
**Pull request by panagiotappl**
closes #120

### CODE REVIEW

File: newdle/client/src/components/creation/FinalStep.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default function FinalStep() {
     history.push('/new/success');
   }
 
-  const canSubmit = title.length >= 3 && !submitting;
+  const canSubmit = title.length >= 3 && title.length <= 80 && !submitting;
```
Comment (ThiefMaster) [inline newdle/client/src/components/creation/FinalStep.js]: didn't we want to put a `maxlength` on the input instaed?

File: newdle/schemas.py [hunk 2]
```diff
@@ -50,7 +50,7 @@ class UpdateNewdleSchema(mm.Schema):
 
 
 class NewNewdleSchema(mm.Schema):
-    title = fields.String(validate=lambda x: len(x) >= 3, required=True)
+    title = fields.String(validate=lambda x: 80 >= len(x) >= 3, required=True)
```
Comment (ThiefMaster) [inline newdle/schemas.py]: `validate=validate.Length(min=3, max=80)`

https://webargs.readthedocs.io/en/latest/quickstart.html#validation

File: newdle/client/src/components/creation/FinalStep.js [hunk 3]
```diff
@@ -13,6 +13,8 @@ import client from '../../client';
 import {newdleCreated, setStep, setTitle} from '../../actions';
 import styles from './creation.module.scss';
 
+const MAX_INPUT_LENGTH = '80';
```
Comment (ThiefMaster) [inline newdle/client/src/components/creation/FinalStep.js]: does using a number cause any warnings?

Comment (panagiotappl) [reply newdle/client/src/components/creation/FinalStep.js]: I thought it had to be a string but guess not

**Pull request merged**